invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar.html

```diff
@@ -35,22 +34,28 @@
 {%- if config.get("APP_RDM_RECORD_EXPORTERS") -%}
 {# if no export formats are specified, don't bother showing the box #}
 <div class="ui segment rdm-sidebar exports">
-  <dt><b>{{ _('Export')}}</b></dt>
-  <hr class="thin-line">
-  </hr>
-  <dd class="top-bottom-padded">
-    <div class="ui celled horizontal list separated-list">
+  <h4>{{ _('Export')}}</h4>
+  <hr class="thin-line"/>
+
+  <div class="selection-wrap">
+    <select class="ui selection dropdown export">
       {# dynamically create the list of export formats #}
+      {%- set ns = namespace (initial_val = "") %}
```

Comment:
minor:
```suggestion
      {%- set ns = namespace(initial_val="") %}
```
Didn't know about this. That `namespace` is actually pretty nice.